### PR TITLE
sczbp: feature description

### PR DESCRIPTION
**Pull request by HLasse**

### CODE REVIEW

File: psycop/common/feature_generation/data_checks/flattened/feature_describer_tsflattener_v2.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def parse_predictor_column_name(
     feature_name_regex: str = r"[a-zA-Z]+_(.*?)_(?=fallback|within)",
     fallback_regex: str = r"_fallback_(.+)$",
     time_interval_start_regex: str = r"_within_([0-9]+)",
-    time_interval_end_regex: str = "_to_([0-9]+)_",
+    time_interval_end_regex: str = r"_to_([0-9]+)_",
```
[the remark this answers is not in the document]
Comment (MartinBernstorff) [reply psycop/common/feature_generation/data_checks/flattened/feature_describer_tsflattener_v2.py:41]: The `r"string"` means to treat the string as "raw", e.g. to treat newline characters as strings:
image

This is super helpful for regex, though! Still support the change 👍

**Comment by MartinBernstorff** [comment]
@HLasse Do you need anything from us here?  :-)

**Comment by HLasse** [comment]
Nono, all good - I'll merge after submission. Thanks <3

**Comment by MartinBernstorff** [comment]
@HLasse Hope it's OK I removed the evergreen label, otherwise feel free to re-add 👍

**Comment by HLasse** [comment]
Once I'm back on Overtaci I'll get it merged 👍

**Pull request merged**